local-tests/README.md

```diff
@@ -0,0 +1,118 @@
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `MAX_ATTEMPTS`           | Each test is executed in a loop with a maximum number of attempts specified by `devEnv.processEnvs.MAX_ATTEMPTS`.                                                        |
+| `NETWORK`                | The network to use for testing, which can be one of the following: `LIT_TESTNET.LOCALCHAIN`, `LIT_TESTNET.MANZANO`, or `LIT_TESTNET.CAYENNE`.                            |
+| `DEBUG`                  | Specifies whether to enable debug mode.                                                                                                                                  |
+| `REQUEST_PER_KILOSECOND` | To execute a transaction with Lit, you must reserve capacity on the network using Capacity Credits. These allow a set number of requests over a period (default 2 days). |
+| `WAIT_FOR_KEY_INTERVAL`  | Wait time in milliseconds if no private keys are available.                                                                                                              |
+| `TIME_TO_RELEASE_KEY`    | Time to wait before releasing the key after requesting it.                                                                                                               |
+| `RUN_IN_BAND`            | Run all tests in a single thread.                                                                                                                                        |
+| `RUN_IN_BAND_INTERVAL`   | The interval in milliseconds to run the tests in a single thread.                                                                                                        |
+| `LIT_RPC_URL`            | The URL of the Lit RPC server. If running locally on Anvil, it should be 'http://127.0.0.1:8545'.                                                                        |
+| `LIT_OFFICIAL_RPC`       | The URL of the official Lit RPC server, usually 'https://chain-rpc.litprotocol.com/http' but can be changed if needed.                                                   |
+
+# Writing a test
+
+Writing a test is the same as writing any other code, except that you must throw an error if any occur. There are no assertion libraries, so all tests are written using basic `if-else` statements.
+
+In the test function, a `devEnv` variable will automatically be added as the first parameter to your function.
+
+## Using the devEnv API in the test
+
+```ts
+export const testExample = async (devEnv: TinnyEnvironment) => {
+
+  // ========== Enviorment ==========
+  // This test will be skipped if we are testing on the Cayenne network
+  devEnv.setUnavailable(LIT_TESTNET.CAYENNE);
```

Comment:
not with capacity delegation auth with when it's not available on cayenne, it will simply throw an `IGNORE` error which would be classified as a `SKIP`